components/BuildPage/build-page.component.tsx

```diff
@@ -84,7 +84,9 @@ function BuildPage({ build }: IBuildPageProps): JSX.Element {
                 <br />({formatSince(build.updatedAt)})
               </SC.AsideSubGroup>
             </AsideGroup>
-            <AsideGroup title="Categories">{build.metadata.type}</AsideGroup>
+            <AsideGroup title="Categories">
+              {build.metadata.categories}
```

Comment:
is this not an array of strings?

if so: `build.metadata.categories.map(category => <div>{category}</div>)`

Reply:
Yea it is, there was some syntax I was unfamiliar with. Will address.